android/app/src/main/res/layout/password.xml

```diff
@@ -1,27 +1,84 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.constraintlayout.widget.ConstraintLayout
+<LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:orientation="vertical">
 
-    <EditText
-        android:id="@+id/etPassword"
-        android:layout_width="0dp"
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="8dp"
-        android:layout_marginEnd="24dp"
         android:layout_marginLeft="24dp"
-        android:layout_marginRight="24dp"
-        android:layout_marginStart="24dp"
-        android:layout_marginTop="8dp"
-        android:ems="10"
-        android:inputType="textPassword"
-        android:imeOptions="actionDone"
-        app:layout_constraintBottom_toBottomOf="parent"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.0"/>
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+        android:layout_marginRight="24dp">
+
+        <TextView
+            android:id="@+id/tvTitle"
+            style="?android:windowTitleStyle"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/fab_dialog_margin_right"
+            android:layout_marginRight="@dimen/fab_dialog_margin_right"
+            android:layout_marginTop="20dp"
+            android:maxLines="3"
+            android:singleLine="false"
+            android:text="@string/lorem"
+            app:layout_constraintEnd_toStartOf="@+id/btnRename"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"/>
+
+        <com.google.android.material.floatingactionbutton.FloatingActionButton
+            android:id="@+id/btnRename"
+            style="@style/FAB.Dialog"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginBottom="@dimen/fab_dialog_margin_bottom"
+            android:layout_marginEnd="@dimen/fab_dialog_margin_right"
+            android:layout_marginRight="@dimen/fab_dialog_margin_right"
```

Comment:
Actually the rightmost button should have no right margin, e.g. see transaction_detail.xml. Sorry for the misleading name: this should be renamed to something like `fab_dialog_margin_inner`. 

Then for consistency, `fab_inner_margin_right`, which serves a similar purpose, should be renamed to `fab_main_margin_inner`.

Reply:
Ok, done.

Reply:
I notice that the dropshadow on the button gets cut off somewhat noticeably when the right margin is set to zero.  This is a problem on the transaction_detail.xml layout too.  It's also cut off slightly on the bottom margin too.  Is it worth adjusting the design to accommodate this?

Reply:
Good point, but let's deal with it separately (#1726 ).